ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -238,6 +238,10 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
+- as: e2e-openstack-ipi
```

Comment:
name? e2e-openstack-ipi makes logs look funny: 

"e2e-openstack-ipi-ipi-rest-of-pod-name"

Reply:
Duplicate `ipi` is expected

Reply:
It is going to be best to remove this addition altogether and merge the step codes first. 
Right now we don't need to run ipi test, but we do need to develop other tests which will use  the same steps.